lib/wallet/open_channel.dart

```diff
@@ -43,8 +43,9 @@ class _OpenChannelState extends State<OpenChannel> {
   @override
   Widget build(BuildContext context) {
     final bitcoinBalance = context.watch<BitcoinBalance>();
-    final maxTakerChannelAmount =
-        bitcoinBalance.confirmed.asSats.clamp(0, OpenChannel.maxChannelAmount);
+    final maxTakerChannelAmount = bitcoinBalance.pendingExternal.asSats != 0
+        ? bitcoinBalance.pendingExternal.asSats.clamp(0, OpenChannel.maxChannelAmount)
+        : bitcoinBalance.confirmed.asSats.clamp(0, OpenChannel.maxChannelAmount);
```

Comment:
🔧 What about `pendingInternal` (change outputs)? I think you _always_ want to use `confirmed + pendingInternal + pendingExternal` to determine the `maxTakerChannelAmount`.

Interestingly, we have a `total` method on `BitcoinBalance` which includes them, but it also includes `pendingChannelClose` which would **not** be safe to use here, as those UTXOs are not tracked by our on-chain wallet yet!

In conclusion, my suggestion would be:

```
final maxTakerChannelAmount = (bitcoinBalance.confirmed.asSats + bitcoinBalance.pendingInternal.asSats + bitcoinBalance.pendingExternal.asSats).clamp(0, OpenChannel.maxChannelAmount);
```

Reply:
😅 My if-or did not make any sense, of course one has to sum them up 🤦‍♂️